user/urls.py

```diff
@@ -1,10 +1,17 @@
 from django.urls import path
-from user.views import CreateUserView, CreateTokenView, ManageUserView
+from user.views import CreateUserView, ManageUserView
+from rest_framework_simplejwt.views import (
+    TokenObtainPairView,
+    TokenRefreshView,
+    TokenVerifyView,
+)
 
 app_name = "user"
 
 urlpatterns = [
     path("register/", CreateUserView.as_view(), name="create"),
-    path("login/", CreateTokenView.as_view(), name="login"),
+    path("token/", TokenObtainPairView.as_view(), name="token_obtain_pair"),
```

Comment:
```suggestion
    path("token/", TokenObtainPairView.as_view(), name="token-obtain-pair"),
``` use '-' instead of '_' here and all of link name